types/node/crypto.d.ts

```diff
@@ -227,7 +227,8 @@ declare module "crypto" {
     interface Decipher extends NodeJS.ReadWriteStream {
         update(data: Binary): Buffer;
         update(data: string, input_encoding: HexBase64BinaryEncoding): Buffer;
-        update(data: Binary, input_encoding: any, output_encoding: Utf8AsciiBinaryEncoding): string;
+        update(data: Binary, output_encoding: HexBase64BinaryEncoding): string;
```

Comment:
is it intentional to remove `input_endcoding` here? based on docu I would expect that input_encoding needs to be given (but `undefined` is allowed).

Reply:
I still think that this overload is not correct - at least the name of the second argument.

Reply:
I think this change is wrong as it's no longer possible to specify only `output_encoding`. The `any` as second arg is correct as node simply ignores it in case first arg is a `Buffer`.

Reply:
not related to your changes: I'm not sure sure if the limitations of encodings to `HexBase64BinaryEncoding` / `Utf8AsciiBinaryEncoding` is correct as it seems that `output_endcoding` `hex` works also.

Reply:
Not going to touch this one for now.

Reply:
I still don't understand the change as it seems it's no longer possible to specify only `output_encoding`.

Reply:
Oh, I might've read the docs wrong, does `output_encoding` take precidence over `input_encoding` and/or should `input_encoding` be optional?
Sorry the docs are very vague here.

Reply:
`output_encoding` is always the third argument. If you don't want to specify `input_encoding`  you can pass `undefined` (utf8 is used then).
In case first arg is a `Buffer` `input_encoding` is ignored so you can effective pass `any`.

Reply:
I think I got it right now? I made `inputEncoding: void` when the data arg is a `Buffer`-like, I think that's most sensible.
If you still think that's wrong please just post what you think is right and I will insert it, this sparse argument stuff is making my head spin today 🤢

Reply:
I think it's correct now. Maybe people will complain as they pass a `string` (which is ignored then) which is no longer allowed now. In that case we have to put back the `any`.